pulser-core/pulser/noise_model.py

```diff
@@ -466,3 +508,50 @@ def from_abstract_repr(obj_str: str) -> NoiseModel:
                 obj_str
             )
         )
+
+    @staticmethod
+    def generate_detuning(
```

Comment:
The name could be more descriptive (and private), e.g.
```suggestion
    def _generate_detuning_fluctuations(
```

Reply:
I use non-private function `generate_detuning_fluctuations` to be able to call it later in `class Hamiltonian`

Reply:
done. it's private